examples/deepspeed_mii/Readme.md

```diff
@@ -0,0 +1,66 @@
+# Running Stable diffusion model using Microsoft DeepSpeed-MII in Torchserve.
+
+This document briefs on serving HG Stable diffusion model with Microsoft DeepSpeed-MII in Torchserve. With DeepSpeed-MII there has been significant progress in system optimizations for DL model inference, drastically reducing both latency and cost.
+
+[Model Paper](https://arxiv.org/abs/2112.10752)
+
+### Step 1: Download model
+
+Login into huggingface hub with token by running the below command
+
+```bash
+huggingface-cli login
+```
+paste the token generated from huggingface hub.
+
+```bash
+python Download_deepseed_mii_models.py --model_path downloaded_model --model_name CompVis/stable-diffusion-v1-4 --revision main
+```
+The script prints the path where the model is downloaded as below.
+
+`downloaded_model/models--bert-base-uncased/snapshots/5546055f03398095e385d7dc625e636cc8910bf2/`
+
+#### Run Stable Diffusion model with DeepSpeed-MII
+
+```bash
+python deepspeed_mii_stable_diffusion.py --model_path downloaded_model/models--bert-base-uncased/snapshots/5546055f03398095e385d7dc625e636cc8910bf2/ --prompt "a dog chaing a cat"
```

Comment:
does this  "deepspeed_mii_stable_diffusion.py" use torchserve? if not can you please clarify the purpose of keeping it hear.